subprojects/test-runner/instrumentation-tests/src/main/kotlin/com/avito/instrumentation/InstrumentationTestsTask.kt

```diff
@@ -83,6 +83,9 @@ public abstract class InstrumentationTestsTask @Inject constructor(
     @Input
     public val useInMemoryReport: Property<Boolean> = objects.property()
 
+    @Input
+    public val fetchLogcatForIncompleteTests: Property<Boolean> = objects.property()
+
```

Comment:
How about grouping experimental toggles in an abstraction?
We could pass it as one container on different levels.

Reply:
i thought about it;
different flags goes to different abstractions so it's probably make sense to create multiple `flag bundles` based on it's destination. For example some of the flags goes to executor, some to reports.

Maybe it's not a big deal and we can move around full blob of experimental flags everywhere.